helm/chart-operator/templates/rbac.yaml

```diff
@@ -160,7 +11,7 @@ subjects:
     namespace: {{ tpl .Values.resource.default.namespace . }}
 roleRef:
   kind: ClusterRole
-  name: {{ tpl .Values.resource.default.name  . }}
+  name: cluster-admin
```

Comment:
The security model changes so now chart-operator needs to be cluster-admin instead of tiller.

Reply:
who create `cluster-role` ?